### ISSUE TITLE
Crawling by # stops at 200

### ISSUE THREAD

**Issue by DGiaco** [huaying/instagram-crawler#46]
Hi! I want to use this crawler to crawl by # and retrieve all the information about a post (num. likes , num. comments...). The original version of the crawler does not allow to do that, so I modified the source code. I succeed in retrieve all the information I need but the crawler stops at 200 posts.
Do you have any insights about that?
Thank you!

**Comment by huaying** [comment]
Yes, I noticed that. It seems like Instagram now have a rate limit policy.

**Comment by DGiaco** [comment]
Thank you very much for the support! In your opinion is possible to crawl
with a "temporal sliding window" 200 post at the time starting from the
newst to the oldest?

Il Gio 16 Mag 2019, 19:29 Hua-Ying Tsai <notifications@github.com> ha
scritto:

**Comment by herewyp** [comment]
Try to add command sleep() after several fetchings to ensure no more than 200 requests for one username are sent per hour. It works well for me but it really takes a long time to crawl :(

fetching: 100%|███████████████████████████| 1000/1000 [5:45:56<00:00,  5.02s/it]
